app/templates/components/widgets/forms/location-input.hbs

```diff
@@ -30,7 +30,7 @@
     </div>
     <div class="column">
       {{#g-map lat=lat lng=lng zoom=zoom class='google-maps' address=combinedAddress as |context|}}
-        {{g-map-address-marker context address=combinedAddress onLocationChange=(action 'onLocationChangeHandler' lat lng)}}
+        {{g-map-address-marker context address=combinedAddress onLocationChange=(action 'onLocationChangeHandler')}}
```

Comment:
@mrsaicharan1 Why you removed the lat and lag variables here

Reply:
restored